package.json

```diff
@@ -108,11 +106,12 @@
     "typescript": "^3.9.7",
     "url-loader": "^4.1.0",
     "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12",
+    "webpack-cli": "^4.2.0",
     "webpack-merge": "^4.2.2"
   },
   "peerDependencies": {
-    "webpack": "^4.0.0 || ^5.0.0"
+    "webpack": "^4.0.0 || ^5.0.0",
+    "webpack-cli": "^4.0.0"
```

Comment:
We should avoid this, and using only `peerDependenciesMeta`, because many boilerplates uses webpack-dev-server without CLI, so we will break them

Reply:
Most package managers now support `peerDependenciesMeta`. WIthout `peerDependencies`, wouldn't it be impossible to enforce the version of `webpack-cli`?

Reply:
Yes, most of supported, so we need to use it, npm@7 automatically install `peerDependencies` :facepalm: But in some cases we don't need `webpack-cli`, for example CRA just use our server, but don't use `webpack-cli`